.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules/
 *.img.xz
 *.img.bz2
 *.log
+.DS_Store
```

Comment:
I stopped to add these files to every repository individually and added them to my systems global gitignore instead. Still not against this, but maybe at all while at it? https://github.com/ebeshero/DHClass-Hub/blob/master/.gitignore